src/pages/Markets/constants.ts

```diff
@@ -7,3 +11,85 @@ export enum MARKETS_CATEGORIES {
   ONE_CLICK_DEFI = 'oneClickDefiAssets',
   THORCHAIN_DEFI = 'thorchainDefi',
 }
+
+export const sortOptionsByCategory: Record<MARKETS_CATEGORIES, SortOption[] | undefined> = {
```

Comment:
Then this guy just becomes something like:

```suggestion
export const sortOptionsByCategory: Record<MARKETS_CATEGORIES, SortOptionsKeys[] | undefined> = {
```

Reply:
Done in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)